src/screens/users/list/Users.tsx

```diff
@@ -17,7 +17,12 @@ import Utils from '../../../utils/Utils';
 import BaseAutoRefreshScreen from '../../base-screen/BaseAutoRefreshScreen';
 import computeStyleSheet from './UsersStyle';
 
-export interface Props extends BaseProps {}
+export interface Props extends BaseProps {
+  select?: ItemsListTypes;
+  modal?: boolean;
+  onUserSelected?: (selectedUsers: User[]) => void;
```

Comment:
To rename selectionMode, isModal, onUsersSelected.

Reply:
onUserSelected is singular because it's triggered every time we select one user

Reply:
If you pass an array, it's plural.